plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/payment-methods-selection/style.scss

```diff
@@ -125,10 +125,10 @@
 
 	.woocommerce-recommended-payment-methods {
 		// Width to fill available space.
-		width: 100%;
-		width: -webkit-fill-available; /* Mozilla-based browsers will ignore this. */
-		width: fill-available;
-		width: -moz-available; /* WebKit-based browsers will ignore this. */
+		width: 100% !important;
```

Comment:
Unfortunately, Jetpack applies a class early in the body class list, making it difficult to override through standard CSS rules. As a result, the only reliable solution is to use `!important`.